src/modules/auth/token.service.ts

```diff
@@ -114,18 +118,14 @@ export class TokenService {
    * @param origin Origin specified for token
    * @param req Http request object being authenticated with token
    */
-  matchOriginAgainstRequest(origin: string, req: Request) {
+  isLoginOriginMatchesRequestOrigin(origin: string, req: Request): boolean {
     const isBrowserRequestFromAuthenticatedOrigin =
       origin === req.headers['origin'];
     const isServerRequestOrGETFromSameDomain =
       req.headers['origin'] === undefined;
-    if (
-      !isBrowserRequestFromAuthenticatedOrigin &&
-      !isServerRequestOrGETFromSameDomain
-    ) {
-      throw new UnauthorizedException(
-        `Token origin ${origin} does not matches request origin ${req.headers['origin']}`
-      );
-    }
```

Comment:
Why the removal of the exception throwing?

Reply:
This is because we are not always throw Unauthorized, if login origins are not matches. `/auth/status` in this case still returns status 200 just with `null` user. So I changed `matchOriginAgainstRequest` to `isLoginOriginMatchesRequestOrigin` to be responsible only for matching and letting client decide how to respond to result of matching